contents/docs/how-posthog-works/recordings-ingestion.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Session recordings ingestion
+---
+
+## Overview
```

Comment:
I feel like we might want to have a disclaimer at the top that this is targeted to PostHog employees. It feels very "internal" rather than being a digestible technical article. 

I still think it should be public, I just don't know if it will be confusing to people who come to see "how do recordings work" and they get all of this...